presto-google-sheets/src/main/java/io/prestosql/plugin/google/sheets/SheetsMetadata.java

```diff
@@ -136,16 +133,15 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
     @Override
     public List<SchemaTableName> listTables(ConnectorSession session, Optional<String> schemaName)
     {
-        if (schemaName.isEmpty()) {
-            throw new PrestoException(SHEETS_UNKNOWN_SCHEMA_ERROR, "Schema not present - " + schemaName);
-        }
-        if (schemaName.get().equalsIgnoreCase("information_schema")) {
-            return ImmutableList.of();
+        ImmutableList.Builder<SchemaTableName> builder = ImmutableList.builder();
```

Comment:
my understanding is that currently the only "default" schema is supported (apart from information schema). The "sheetsClient" doesn't seem to understand the concept of a "schema", but it is being repeatedly invoked in a loop iterating over schema objects. So there's kind of an assumption that SCHEMAS has size=1. 

I feel we could make the logic more explicit by:

- either making `sheetsClient#getTableNames` aware of schema names
- or having `SheetsMetadata#listTables` method explicitly check that "default" is the desired schema, before invoking `sheetsClient#getTableNames()`. 

Thoughts?